core/plugins/org.polarsys.capella.core.sirius.analysis/description/oa.odesign

```diff
@@ -418,14 +418,14 @@
           <ownedTools xsi:type="tool:ToolDescription" documentation="Insert/Remove Operational Activities. Create or delete graphical objects corresponding to already existing Operational Activities. Removing an Operational Activity graphical object from the diagram does not delete the Operational Activity from the model. This tool does not modify the semantic model." name="functions" label="%oa.oaib.functions" iconPath="/org.polarsys.capella.core.sirius.analysis/icons/full/obj16/ShowHide.gif">
             <element name="element"/>
             <elementView name="elementView">
-              <subVariables xsi:type="tool:AcceleoVariable" name="visibleFunctions" computationExpression="aql:self.void2Null(elementView.getDisplayedAbstractFunctions()->select(getDisplayedAbstractFunction | getDisplayedAbstractFunction.allocationBlocks->filter(cs::Component)->select(c | c.isActor())->size() = 0))"/>
-              <subVariables xsi:type="tool:AcceleoVariable" name="visibleFunctionViews" computationExpression="aql:self.void2Null(elementView.getDisplayedAbstractFunctionViews()->select(getDisplayedAbstractFunctionView | getDisplayedAbstractFunctionView.target.allocationBlocks->filter(cs::Component)->select(c | c.isActor())->size() = 0))"/>
+              <subVariables xsi:type="tool:AcceleoVariable" name="visibleFunctions" computationExpression="aql:self.void2Null(elementView.getDisplayedAbstractFunctions())"/>

```

Comment:
Its now consistent with SDFB / LDFB and PDFB

Reply:
Do we know why we previously chose to filter activities allocated to actors?